test/checkTest.spec.ts

```diff
@@ -0,0 +1,23 @@
+import Vorpal from "vorpal";
+import * as checkTest from "../src/commands/checkTest";
+import { CheckUtils } from "../src/classes/check-utils";
+import { CheckSet } from "../src/interfaces/types";
+
+it('exports itself', () => {
+  const vorpal = new Vorpal();
```

Comment:
What does this actually test?

Reply:
Accessibility, and its exported functionality for Vorpal. In this case, function Action(). CheckTest is currently a stub function that will run checks for projects in the future. The .spec will have more functionality in another issue.